share_app/src/main/AndroidManifest.xml

```diff
@@ -5,6 +5,7 @@
     <uses-permission android:name="android.permission.CHANGE_WIFI_STATE" />
     <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+    <uses-permission android:name="android.permission.WRITE_SETTINGS" />
```

Comment:
Why is this permission required?

Reply:
Whenever we create new WifiConfiguration it requires this permission.